Assets/Scripts/Entity.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections;
+using UnityEngine;
+
+public class Entity : MonoBehaviour, IDamagable
+{
+    [SerializeField]
+    public float initialHealth = 100;
+    public float health { get { return _health; } set { _health = value; } }
```

Comment:
`public float health { get ; private set; }`

_health is not needed.

Reply:
Done: https://github.com/cjh15hub/terribletweeters/pull/1/commits/81f82db7e00559acff3871b77076b9bdb9bcaa43#diff-8446b20d766823004b26f3f1b33f5831cf745c6d4a21828ccea0479ff1c54ce9R8

Reply:
Done: https://github.com/cjh15hub/terribletweeters/pull/1/commits/81f82db7e00559acff3871b77076b9bdb9bcaa43#diff-8446b20d766823004b26f3f1b33f5831cf745c6d4a21828ccea0479ff1c54ce9R8